iothub/device/samples/readme.md

```diff
@@ -72,6 +77,7 @@ You need to clone the repository or download the sample (the one you want to try
 [m-message-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/iot-hub/Samples/module/ModuleSample
 [lnk-setup-iot-hub]: https://aka.ms/howtocreateazureiothub
 [lnk-manage-iot-device]: https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md#create-new-device-in-the-iot-hub-device-identity-registry
+[lnk-setup-iot-edge]: https://github.com/Azure/azure-iot-sdk-csharp/blob/tmahmood/update-sample-repo/iothub/device/samples/getting%20started/EdgeModuleMessageSample/ReadMe.md
 [d-read-d2c-messages-sample]: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/getting%20started/ReadD2cMessages
```

Comment:
this should point to main branch

Reply:
I agree it shouldn't point to tmahmood's branch, but not main. I'd say previews/v2 until we merge that to main.

Reply:
sure, pointing it to previews/v2 for now.